nrf_wifi/fw_if/umac_if/inc/fw/host_rpu_sys_if.h

```diff
@@ -630,6 +630,12 @@ struct rpu_lmac_stats {
 	unsigned int scan_abort_complete;
 	/** Number of internal buffer pool null counts */
 	unsigned int internal_buf_pool_null;
+	/** Lockup recovery event*/
+	unsigned int rpuLockupEvent;
+	/** Lockup revovery count*/
+	unsigned int rpuLockupCnt;
+	/** Lockup recovery state*/
+	unsigned int rpuLokcupRecoveryDone;
```

Comment:
We will not be able to modify the existing structure due to ROM firmware. Please add new Command/event to get additional debug data.